cmd/mysql-operator/app/mysql_operator.go

```diff
@@ -28,7 +28,7 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
 
-	options "github.com/oracle/mysql-operator/cmd/mysql-operator/app/options"
+	opratoropts "github.com/oracle/mysql-operator/pkg/options/operator"
```

Comment:
Minor typo here (opratoropts => operatoropts)

Reply:
Thanks for the review.